src/globus_sdk/experimental/globus_app/app.py

```diff
@@ -165,6 +157,39 @@ def _resolve_client_info(
                 "login_client is required."
             )
 
+    def _finalize_token_storage(self) -> None:
```

Comment:
Not immediately clear to me that this full listing should live in GlobusApp itself. I'd love to push it into the "validing token storage" space somehow.

Reply:
I didn't move it very far, but it is now in a dedicated helper method.

I'm resolving, but open to further discussion.